nautobot/core/templates/about.html

```diff
@@ -0,0 +1,67 @@
+{% extends 'base.html' %}
+{% load helpers %}
+{% load registry %}
+
+{% block header %}
+    {{ block.super }}
+    {% if new_release %}
+        {# new_release is set only if the current user is a superuser or staff member #}
+        <div class="alert alert-info text-center" role="alert">
+            <i class="mdi mdi-information-outline"></i>
+            A new release is available: <a href="{{ new_release.url }}">Nautobot v{{ new_release.version }}</a> |
+            <a href="https://docs.nautobot.com/projects/core/en/v{{ new_release.version }}/installation/upgrading/">Upgrade instructions</a>
+        </div>
+    {% endif %}
+{% endblock %}
+
+
+{% block content %}
+    <h1>About Nautobot</h1>
```

Comment:
Should any part of this page make use of the configured `BRANDING_TITLE` setting?

Reply:
Jason and I talked about maybe included a statement when you have set `BRANDING_TITLE`, but otherwise the intent is to be explicit about Nautobot on this one page.

Reply:
Correct.